app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -355,8 +342,9 @@ private fun FavoriteParkingCard(parking: Parking, index: Int, onRemove: () -> Un
         title = { Text(stringResource(R.string.view_profile_screen_remove_favorite_dialog_title)) },
         text = {
           Text(
-              stringResource(R.string.view_profile_screen_remove_favorite_dialog_message)
-                  .format(parking.optName))
+              stringResource(
+                  R.string.view_profile_screen_remove_favorite_dialog_message,
+                  parking.optName ?: ""))
```

Comment:
There is a `default_parking_name` String that exists in `strings.xml` that you can use instead of an empty string for an unnamed parking.